util/src/main/java/org/datacommons/util/TmcfCsvParser.java

```diff
@@ -375,6 +368,14 @@ private String newUUID() {
   }
 
   private void addLog(Debug.Log.Level level, String counter, String message) {
-    logCtx.addEntry(level, counter, message, csvFileName, csvParser.getCurrentLineNumber());
+    logCtx.addEntry(level, counter, message, csvFileName, getCurrentLineNumber());
+  }
+
+  // Returns the current line in the CSV.
+  private long getCurrentLineNumber() {
+    // We track the line number using a private variable instead of using
+    // csvparser.getCurrentLineNumber() because that function does not increment
+    // the line number for the last line that does not end in a newline.
```

Comment:
I would move this comment to the definition of the variable.

Reply:
done.